### PR TITLE
Issue #13345: Enable RedundantModifierCheckExampleTest

### PR DESCRIPTION
**Pull request by mahfouz72**
part of #13345, needed for #15376

### CODE REVIEW

**Comment by mahfouz72** [comment]
github, generate site

**Comment by mahfouz72** [comment]
@romani It will be good if we can merge this with a single review, to be able to extend examples with the new property in the other PR. I didn't want to extend them without enabling the ExampleTest

**Review by romani — APPROVED** [review]
Code is ok

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/modifier/redundantmodifier/Example1.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    }
+  }
+
+  // violation below, 'Redundant 'abstract' modifier'
+  abstract interface I {
+    public abstract void m();
+    // 2 violations above:
+    //                   'Redundant 'public' modifier'
+    //                   'Redundant 'abstract' modifier'
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/modifier/redundantmodifier/Example1.java:28]: Please make indentation to be 4 space to make code less wide and fit phone screen:)

**Review by romani — APPROVED** [review]
Merging quicker to unblock other PR
Please fix in follow up PR

**Comment by romani** [comment]
@nrmancuso , @rnveach , fyi.

**Pull request merged**